### PR TITLE
Add favicon

### LINKED ISSUE #1071: Add Favicon for the gnomAD browser
Related slack message: https://atgu.slack.com/archives/CNLMF0JLV/p1674904304614369

Add a Favicon to gnomAD.

When users have many open tabs, a Favicon helps them see at a glance which ones are gnomAD.

This could be a single blue peak in the style of the depth of reads chart for a gene/region, or a gnome (not sure if the gnome comment is a joke or not).

### PR DESCRIPTION
**Pull request by rileyhgrant**
Resolves #1071 

Adds a Favicon to gnomAD, per slack discussion [here](https://atgu.slack.com/archives/CNLMF0JLV/p1674904304614369).

Per Matts suggestion, this takes the form of the first two peaks of the depth of reads graph from the main logo on the home page. I thought that padding around the edges and the inclusion of the gnomad 'g' made it more distinguishable when small, but am hoping for feedback and suggestions.

This lets one keep track of gnomAD tabs in a sea of tabs.

- example 1
  ahhhh

- example 2
  Screenshot 2023-02-21 at 10 51 26

The current favicon currently set to be used is: 
gnomad-favicon-3

---
## Other options:

- Open to any suggestions

- option 2
  gnomad-favicon-1
 
- option 3
  gnomad-favicon-2

### CODE REVIEW

**Comment by rileyhgrant** [comment]
Also - this currently includes a few other favicon sizes (different ones for mobile based on OS, the old `favicon.ico` as I used some site to generate the favicons based off the `png`. 

These are not currently loaded by the webpack loader, but if desired to have all those optimized favicon sizes we could hook up the generated web manifest and get that going.

**Comment by rileyhgrant** [comment]
@mattsolo1, @phildarnowsky 

Requesting review from both you primarily just to get input on what the Favicon should be / if it should be changed. After a round of input from the gnomad-browser-devs I'll post in the gnomad-browser slack channel for a wider pool of input.

File: browser/src/index.html [hunk 1]
```diff
@@ -1,25 +1,42 @@
-<!doctype html>
+<!DOCTYPE html>
```
Comment (phildarnowsky-broad) [inline browser/src/index.html:1]: I'm guessing that most of the changes here were automatic reformatting by your editor or a linter, and the favicon line is the only new part. If that's the case, it's often good, in a situation like this, to have a separate commit where you lint the existing code to make it pretty, before you dig in and make the real changes. That way it's clear what changes actually are functional, as opposed to just aesthetic.

Comment (phildarnowsky-broad) [reply browser/src/index.html:1]: However in this case, it's probably OK to leave as-is. Something to keep in mind for next time though.

**Comment by phildarnowsky-broad** [comment]
@rileyhgrant /cc @mattsolo1 please note that I have a separate Github account for Broad work, @phildarnowsky-broad. @phildarnowsky is my personal account.

**Comment by phildarnowsky-broad** [comment]
@rileyhgrant the code looks OK to me, and I think option #1, i.e. the one you've already got in here, is my favorite version of the three. Whichever design we go with, I think retaining the "g" would be good, since two blue peaks by themselves could mean most anything.

File: browser/src/index.html (hunk 1 above)
Comment (rileyhgrant) [reply browser/src/index.html:1]: Ohh thats a very good call. I can go back and clean this up later today. Thanks!

**Comment by rileyhgrant** [comment]
> @ rileyhgrant /cc @ mattsolo1 please note that I have a separate Github account for Broad work, @ phildarnowsky-broad. @ phildarnowsky is my personal account.

Oops, yeah in theory I knew that re: requesting reviews, but didn't think to double check when I tag you in comments. I'll be more careful about that in the future

**Comment by rileyhgrant** [comment]
Oops, I didn't review comments on the initial issue, and thus missed Matt's suggestion of using a favicon webpack plugin to simplify the manifest and sizing of favicons for different devices.

I added a commit that adds that dependency and uses it, and if that is the overall preferred way (I prefer it) I will drop the commit that adds the favicon in a different way.

**Comment by mattsolo1** [comment]
Have to say I'm not in love with the "g" in the icon. The picture is nice when it is big but when you shrink it down to icon size, it doesn't seem to work as well. Plus the "g" in on it's own reminds me a bit too much of Google.

Any other ideas? I tried a few on my own in photoshop but couldn't come up with anything good.

**Comment by mattsolo1** [comment]
Can you add the option 2 svg to the pull request so I can try on my machine?

**Comment by rileyhgrant** [comment]
> Can you add the option 2 svg to the pull request so I can try on my machine?

Oop yeah, will do. I'll also add a square screenshot genome/exome coverage from some gene, in case that might look decent at a small scale

**Comment by mattsolo1** [comment]
Here are the 3 icons in 1 view. I can't see much of a difference between the two "g" options when the icons are tiny.

I think I like either tab 2 or 3 in this screenshot.

image

**Comment by mattsolo1** [comment]
@rileyhgrant could you please try two more things

1. Put the coverage-only (tabs 2 and 3 in the screenshot) icons on a transparent background
2. Enclose both coverage-only icons in a circle with color `#fafafa` (same as gnomAD's background color). For example look at Github's favicon or Google's favicon, who enclose their logos in a circle

**Comment by rileyhgrant** [comment]
Here are a few screenshots with some more options

Screenshot 2023-02-28 at 12 33 38

Screenshot 2023-02-28 at 12 29 39

More crowded screenshot
Screenshot 2023-02-28 at 12 30 20


Of these, I'm most partial to the blank background 2 peaks of a exome depth of reads chart, or the original gnomad-background colored exome depth of reads with the 'g'. I think the lowercase g in the gnomad font could help a user distinguish the icon at a glance, and though google uses a 'G', I don't find it that reminiscent of the gnomad G.

I like the thought of the round background, but I'm not sure it works for any of the iconography we're trying with gnomad. google's G is naturally round, and GitHub fits their very distinguishable icon very clever within the roundness. Many websites use a square or rounded corners square for a favicon (Broad Institute, Strava, Google Calendar).

**Comment by mattsolo1** [comment]
Agreed, the round icons don't work at all. 

> I'm most partial to the blank background 2 peaks of a exome depth of reads chart

By blank you mean transparent? If so I agree.

**Comment by rileyhgrant** [comment]
Screenshots of option 2 from the poll, now as an SVG

Screenshot 2023-03-02 at 09 14 43

Screenshot 2023-03-02 at 09 14 52

**Review by mattsolo1 — APPROVED** [review]

**Pull request merged**